arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,931 @@
+          "description": "The namespace type.",
+          "enum": [
+            "Messaging",
+            "NotificationHub",
+            "EventHub"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceType",
+            "modelAsString": true
+          }
+        }
+      },
+      "x-ms-client-flatten": true,
+      "description": "Properties of the Namespace."
+    },
+    "RegenerateKeysParameters": {
+      "properties": {
+        "Policykey": {
+          "type": "string",
+          "description": "Key that needs to be regenerated .",
+          "enum": [
+            "PrimaryKey",
+            "SecondayKey"
+          ],
+          "x-ms-enum": {
+            "name": "Policykey",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Parameters supplied to the Regenerate Namespace Auth Rule."
+    },
+    "NamespaceCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Namespace data center location."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Namespace tags."
+        },
+        "properties": {
+          "$ref": "#/definitions/NamespaceProperties"
+        }
+      },
+      "required": [
+        "location",
+        "properties"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Namespace operation."
+    },
+    "NamespaceResource": {
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "Kind of the Namespace.",
+          "enum": [
+            "Messaging",
+            "EventHub"
+          ],
+          "x-ms-enum": {
+            "name": "kind",
+            "modelAsString": true
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/ARMSku"
+        },
+        "properties": {
+          "$ref": "#/definitions/NamespaceProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of a Namespace resource."
+    },
+    "ARMSku": {
+      "type": "object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The resource type that this object applies to. Eg: NamespaceType",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "Name",
+            "modelAsString": true
+          }
+        },
+        "Tier": {
+          "type": "string",
+          "description": "The tier of this particular SKU",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "Tier",
+            "modelAsString": true
+          }
+        },
+        "Capacity": {
```

Comment:
is there a max/min constraint on this property that the server enforces? If yes, then you should be specifying that over here. Check this [link](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#patterned-fields-2). Once the page is loaded then scroll up and you will see the set of constraints that one can specify.